packages/react-components/components/SectionHead.v2.tsx

```diff
@@ -0,0 +1,30 @@
+import colors from '@celo/react-components/styles/colors.v2'
+import fontStyles from '@celo/react-components/styles/fonts.v2'
+import * as React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+interface Props {
+  text: string
+}
+
+function SectionHead({ text }: Props) {
+  return (
+    <View style={style.container}>
+      <Text style={style.text}>{text}</Text>
+    </View>
+  )
+}
+
+const style = StyleSheet.create({
+  container: {
+    paddingTop: 20,
+    paddingBottom: 8,
+    paddingHorizontal: 16,
+  },
+  text: {
+    ...fontStyles.h2,
+    color: colors.dark,
+  },
+})
+
+export default SectionHead
```

Comment:
I've updated `SectionHeadNew` in a previous master that's already been merged:
https://github.com/celo-org/celo-monorepo/blob/ee5242cb59ef61a35718d76c8dc2447d0c71c7a0/packages/react-components/components/SectionHeadNew.tsx
We can use that one now.

Reply:
On the Gold tab they have a different styling, so I will rename to `SectionHeadGold.tsx` for now.

Reply:
Oh ok, all good then 👍